### PR TITLE
add initial NN model

### PR DESCRIPTION
**Pull request by joeddav**
Have an initial NN implemented. I want to add validation loss in training procedure so that we know if models overfit.

### CODE REVIEW

**Comment by JoshFeldman95** [comment]
Yeah good call
